charmcraft.yaml

```
@@ -56,6 +56,19 @@ parts:
 
 config:
   options:
+    create-authorization-policies:
+      type: boolean
+      default: true
+      description: >
+        Automatically create Istio authorization policies for any charm requiresting them over the service mesh 
```

```suggestion
        Automatically create Istio authorization policies for any charm requesting them over the service mesh 
```
Feel free to ignore the suggestion if that was actually a word and my English failed me :D

---

Your English served you better than mine here :)